main.go

```diff
@@ -61,6 +61,6 @@ func main() {
 	router.GET("/albums", getAlbums)
 	router.POST("/add", addAlbum)
 	router.GET("/getAlbumById/:id", getAlbumById)
-	router.DELETE("/deleteById/:name", deleteById)
+	router.DELETE("/deleteByTitle/:name", deleteByTitle)
```

Comment:
You're changing my `title` but your endpoint Param is called `name`. Be consistent

Reply:
Made changes.